packages/gamut/src/ProgressBar/index.tsx

```diff
@@ -63,6 +70,14 @@ export const ProgressBar: React.FC<ProgressBarProps> = ({
         height: `${height}px`,
       }}
     >
+      {stripes && (
+        <Pattern
+          className={styles.stripes}
+          width="100%"
+          height={36}
```

Comment:
Amusing that it works as expected right now, but this feels reasonable?

```suggestion
          height={height}
```